kvssd/test/kvssd_test.cc

```diff
@@ -0,0 +1,224 @@
+#include "gtest/gtest.h"
+#include "../kvssd_hashmap_db_impl.h"
+
+#include <ctime>
+#include <string>
+
+#define NUM_KEYS 1000
+#define NUM_VALUES 1000
+
+using namespace ycsbc;
+
+std::string RandomPrintStr(size_t length)
+{
+
+    std::string randomString;
+    randomString.reserve(length);
+    for (size_t i = 0; i < length; ++i)
+    {
+        randomString += utils::RandomPrintChar();
+    }
+    return randomString;
+}
+
+std::vector<std::string> key = []
+{
+    std::vector<std::string> keys(NUM_KEYS);
+    for (size_t i = 0; i < NUM_KEYS; i++)
+    {
+        keys[i] = "key" + std::to_string(i);
+    }
+    return keys;
+}();
+
+std::vector<std::vector<DB::Field>> value = []
+{
+    srand(static_cast<unsigned>(time(0)));
+    std::vector<std::vector<DB::Field>> values(NUM_VALUES);
+    for (size_t i = 0; i < NUM_VALUES; i++)
+    {
+        values[i] = {
+            {"field" + std::to_string(i) + "_1", "value" + std::to_string(i) + RandomPrintStr(32)},
+            {"field" + std::to_string(i) + "_2", "value" + std::to_string(i) + RandomPrintStr(32)},
+            {"field" + std::to_string(i) + "_3", "value" + std::to_string(i) + RandomPrintStr(32)}};
+    }
+    return values;
+}();
+
+std::vector<DB::Field> output_value;
+
+std::unique_ptr<KVSSD> kvssd(NewKvssdDB());
+
+bool FieldVectorCmp(std::vector<DB::Field> &value1, std::vector<DB::Field> &value2)
+{
+    size_t len1 = value1.size();
+    size_t len2 = value2.size();
+    if (len1 != len2)
+    {
+        return true;
+    }
+    for (size_t i = 0; i < len1; i++)
+    {
+        if (value1[i].name != value2[i].name)
+        {
+            return true;
+        }
+        if (value1[i].value != value2[i].value)
+        {
+            return true;
+        }
+    }
+    return false;
+}
+
+// ----------------------- TEST START ----------------------- //
+
+TEST(READ, Simple)
```

Comment:
```suggestion
TEST(KvssdHashMapDbImplTest, ReadSmall)
```

좀 더 명확하게 테스트 이름을 정해주시면 좋을 것 같습니다.

Reply:
이후 TEST들도 함께 수정해주세요.

Reply:
명시된 내용처럼 수정하겠습니다.